autoreduce_utils/credentials.py

```diff
@@ -8,49 +8,46 @@
 """
 Settings for connecting to the test services that run locally
 """
-import configparser
-
-from autoreduce_utils.settings import CREDENTIALS_INI_FILE
+import os
 from autoreduce_utils.clients.settings.client_settings_factory import ClientSettingsFactory
 
-CONFIG = configparser.ConfigParser()
-CONFIG.read(CREDENTIALS_INI_FILE)
-
-
-def get_setting(section: str, key: str) -> str:
-    """
-    Gets the value of the key from the section.
-    """
-    return str(CONFIG.get(section, key, raw=True,
-                          fallback=''))  # raw=True to allow strings with special characters to be passed
+from dotenv import load_dotenv
```

Comment:
An alternative to doing this in code would be to use [direnv](https://direnv.net/), this has the benefit of being one less dependency for the Python codebase and allows development and production workflows to be closer aligned with one another.

We would just need to document the need to have direnv installed in the developer documentation.